docs: updated table panel visualization (#90310)
grafana/grafana: docs/sources/panels-visualizations/visualizations/table/index.md

{{< docs/play title="Table Visualizations in Grafana" url="https://play.grafana.org/d/OhR1ID6Mk/" >}}

{{< admonition type="note" >}}
Annotations and alerts are not currently supported for tables.
{{< /admonition >}}

## Sort column

Click a column title to change the sort order from default to descending to ascending. Each time you click, the sort order changes to the next option in the cycle. You can sort multiple columns by holding the `shift` key and clicking the column name.

![Sort descending](/static/img/docs/tables/sort-descending.png 'Sort descending')

## Data set selector

If the data queried contains multiple data sets, a table displays a drop-down list at the bottom, so you can select the data set you want to visualize.

> ![Table Panel with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png)

Reviewer: ```suggestion
![Table visualization with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png)
```

{{< docs/play title="Table Visualizations in Grafana" url="https://play.grafana.org/d/OhR1ID6Mk/" >}}

### Key Features:
- **Multi-Mode Support**: Seamlessly visualize time series data, detailed tables, annotations, or raw JSON inputs.
- **Advanced Formatting**: Customize your data presentation with options for date formatting and value formatting.
- **Coloring Options**: Enhance readability and insights by applying custom coloring schemes to your data points.

> - **Diverse Cell Types**: Elevate your data display with a variety of cell types, including: Colored text and backgrounds, Gauges, JSON, Images, Sparklines, and multiple combinations.

Author: Agreed, I was unsure of adding this part myself and now more that you pint out the marketing sound 😛 
If it doesn't make much sense we could remove it completely.

> ### Key Features:
> - **Multi-Mode Support**: Seamlessly visualize time series data, detailed tables, annotations, or raw JSON inputs.
> - **Advanced Formatting**: Customize your data presentation with options for date formatting and value formatting.
> - **Coloring Options**: Enhance readability and insights by applying custom coloring schemes to your data points.
> - **Diverse Cell Types**: Elevate your data display with a variety of cell types, including: Colored text and backgrounds, Gauges, JSON, Images, Sparklines, and multiple combinations.

Reviewer: Yeah, giving it more thought, I don't think this fits here. Let's take it out.
```suggestion
```

## Annotation and alert support

> {{% admonition type="note" %}}
> Annotations and alerts are not currently supported for tables.
> {{% /admonition %}}

Reviewer: A heading must be followed by some body text. I'm okay with making this a note, but it should either be a note without a heading or plain text with a heading. Do you think people need this information enough to keep a heading? If not, then we should remove the heading and leave it as a note. Also just updating the format of the admonition.
```suggestion
{{< admonition type="note" >}}
Annotations and alerts are not currently supported for tables.
{{< /admonition >}}
```

> ## Table visualization functions
> 
> The table visualization provides multiple functions.

Reviewer: I'd like to not add this heading as my plan is to eventually move all the configuration options under a new H2 to group them better. That will make all of these other tasks and features a bit more prominent.
```suggestion
```

> The table visualization itself provides multiple functions.

Reviewer: ```suggestion
The table visualization provides multiple functions.
```

> ### Sort column

Reviewer: ```suggestion
## Sort column
```

Click a column title to change the sort order from default to descending to ascending. Each time you click, the sort order changes to the next option in the cycle. You can sort multiple columns by holding the `shift` key and clicking the column name.

![Sort descending](/static/img/docs/tables/sort-descending.png 'Sort descending')

> ### Data set selector

Reviewer: ```suggestion
## Data set selector
```

> The table panel visualization will display a dropdown list at the bottom if the data queried contains multiple data sets.

Reviewer: ```suggestion
If the data queried contains multiple data sets, a table displays a drop-down list at the bottom, so you can select the data set you want to visualize.
```

> ![Table Panel with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png 'Table Panel with multiple data sets')

Reviewer: Only alt text is required. FYI, please take a look at the [image naming guidelines](https://grafana.com/docs/writers-toolkit/write/image-guidelines/#media-asset-file-naming-conventions) for future reference. But don't worry about updating it for this PR.
```suggestion
![Table Panel with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png)
```

> ## Annotation and alert support
> 

Reviewer: ```suggestion
```

Cells can be displayed as a graphical gauge, with several different presentation types.

{{% admonition type="note" %}}

> The maximum and minimum values of the Gauges will be configured automatically from the smallest and largest values in your whole data set. If you don't want the max/min values to be pulled from the whole data set, you can configure them for each column with field overrides.

Reviewer: ```suggestion
The maximum and minimum values of the gauges are configured automatically from the smallest and largest values in your whole data set. If you don't want the max/min values to be pulled from the whole data set, you can configure them for each column with field overrides.
```

> {{% admonition type="note" %}}
> The maximum and minimum values of the gauges are configured automatically from the smallest and largest values in your whole data set. If you don't want the max/min values to be pulled from the whole data set, you can configure them for each column with field overrides.
> {{% /admonition %}}

Reviewer: ```suggestion
{{< admonition type="note" >}}
The maximum and minimum values of the gauges are configured automatically from the smallest and largest values in your whole data set. If you don't want the max/min values to be pulled from the whole data set, you can configure them for each column with field overrides.
{{< /admonition >}}
```
